docs/API.md

```diff
@@ -31,35 +31,161 @@ The following tags are available to override Datadog specific options:
 
 <h3 id="scope-manager">Scope Manager</h3>
 
-In order to provide context propagation, this library includes a scope manager. A scope is basically a wrapper around a span that can cross both synchronous and asynchronous contexts.
+In order to provide context propagation, this library includes a scope manager.
+A scope is basically a wrapper around a span that can cross both synchronous and
+asynchronous contexts.
 
-For example:
+The scope manager contains 3 APIs available on `tracer.scope()`:
+
+<h4>scope.active()</h4>
+
+This method returns the active span from the current scope.
+
+<h4>scope.activate(span, fn)</h4>
+
+This method activates the provided span in a new scope available in the function
+provided as the second argument. Any asynchronous context created from whithin
+that function will also have the same scope.
 
 ```javascript
-const tracer = require('dd-trace').init({ plugins: false })
-const express = require('express')
-const app = express()
+const tracer = require('dd-trace').init()
+const scope = tracer.scope()
+const span = tracer.startSpan('web.request')
+const promise = Promise.resolve()
+const log = console.log
 
-app.use((req, res, next) => {
-  const span = tracer.startSpan('web.request')
-  
-  tracer.scope().activate(span, () => next())
-})
+scope.activate(span, () => {
+  log(scope.active()) // the span because in new scope
+
+  someFunction() // the span because called in scope
```

Comment:
We probably don't need this comment here since we aren't accessing the original `span`

Reply:
ah, I see why we have it.

Reply:
I wanted to put it near the log statement, but it's not true that the function would always be in that scope so I preferred to put it there.